utils.c

```diff
@@ -617,3 +620,34 @@ pivot_root (const char * new_root, const char * put_old)
   return -1;
 #endif
 }
+
+char * label_mount(const char *opt, const char *mount_label) {
+  if (!mount_label)
+    return strdup_printf(opt);
+
+#ifdef HAVE_SELINUX
+  if (!opt)
+    return strdup_printf ("context=\"%s\"", mount_label);
+
+  if (opt && mount_label)
+	  return strdup_printf ("%s,context=\"%s\"", opt, mount_label);
+#endif
```

Comment:
Yes I need to figure out how to get emacs to do the right thing with this spacing.